06_クラウドインフラ/04_S3を理解する/lifecycle-expiration.tf

```
@@ -0,0 +1,25 @@
+locals {
+  name_expiration = "lifecycle-expiration"
+}
+
+resource aws_s3_bucket lifecycle-expiration {
+  bucket = "${local.name_expiration}"
```

変数展開しなくても動きそうですね
完全に好みの問題ですけど 🤣 
```suggestion
  bucket = local.name_expiration
```